src/main/java/com/mentorship/vineservice/dto/OrderVineDto.java

```diff
@@ -0,0 +1,20 @@
+package com.mentorship.vineservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderVineDto {
+
+    private Long orderId;
```

Comment:
why do you need this field here? Can this class be only a part of order? Have u thought about inner class?